src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.commands;
+
+/**
+ * Sorts the address book.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = "Sorts address book permanently in alphabetical order of names.\n"
```

Comment:
The usage message should be consistent with the other commands.
```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts address book permanently in alphabetical order of names.\n"
```

Reply:
Yes, I missed this. Applied.